### ISSUE TITLE
The output of the siCluster model and the loss function become NAN for any batch size.

### ISSUE THREAD

**Issue by axin20** [dscig/urban_score#1]
Hi, dear authors

I downloaded some of the daylight images according to your codes. And when I run "siCluster.py", the output of model and the loss function become NAN for any batch size. Does it has something to do with the size of the training dataset?  Cause I just use 150 images as input to the siCluster. And the image size is 224*224*3. The training process is as shown in the picture below.  
![1](https://user-images.githubusercontent.com/62279742/88688414-e2609200-d12b-11ea-8179-b26ab46e5670.JPG)
![2](https://user-images.githubusercontent.com/62279742/88688488-fad0ac80-d12b-11ea-94bf-e99480feddb9.JPG)

**Comment by Sungwon-Han** [comment]
A small number of images with a large number of clusters can result in an error (or unexpected behavior) in our code.
Please try reducing the number of clusters or try adding more satellite images.

**Comment by axin20** [comment]
Thanks a lot. That really solves my problem. And there is another question that can you illustrate how the module "night_eval.py" works and how can I get the kr_nightlight.csv file from the codes . Because in repository "data", the corresponding codes are missing and I cannot do the related works with nightlight imagery.

**Comment by DonghyunAhn** [comment]
Thanks for reporting! Now I add the missing code "night_eval.py"
